lib/game_selection.py

```diff
@@ -14,13 +19,28 @@ def select_game_type(self):
         self.ui.display("Choose h for human vs. human game or c for human vs computer")
         input = self.ui.get_user_input()
         if input == "h":
-            current_player = HumanPlayer(self.ui, self.board, "X")
-            next_player = HumanPlayer(self.ui, self.board, "O")
+            marker_player_1 = self.select_a_marker(self.first_player)
+            current_player = HumanPlayer(self.ui, self.board, marker_player_1)
+            marker_player_2 = self.select_a_marker(self.second_player)
+            while marker_player_1 == marker_player_2:
+                self.ui.display("This marker is already used. Choose another one!")
+                marker_player_2 = self.select_a_marker(self.second_player)
+            next_player = HumanPlayer(self.ui, self.board, marker_player_2)
             return [current_player, next_player]
         elif input == "c":
-            current_player = HumanPlayer(self.ui, self.board, "X")
-            next_player = ComputerPlayer(self.board, "O")
+            marker_player_1 = self.select_a_marker(self.first_player)
+            current_player = HumanPlayer(self.ui, self.board, marker_player_1)
+            marker_player_2 = self.select_a_marker(self.second_player)
+            next_player = ComputerPlayer(self.board, marker_player_2)
             return [current_player, next_player]
```

Comment:
`select_game_type ` is a very large method. Could you refactor some of the logic into smaller methods using the [extract method](https://refactoring.guru/extract-method)?

Reply:
done